packages/axios-extension/src/factory.ts

```diff
@@ -85,6 +85,25 @@ export function createForAbapOnBtp(
     return provider;
 }
 
+/**
+ * Create an instance of an ABAP service provider for a Cloud ABAP system
+ *
+ * @params options
+ * @param options.url ABAP Service Instance URL
+ * @returns instance of an ABAP service provider
+ */
+export function createForAbapOnCloud({
```

Comment:
This function is misleading. We already have `createForAbapOnBtp` which is the same as cloud (well, not literally but you know what I mean). So, we should extend the existing function and allow both options, reentrance tickets and oauth

Reply:
Or have to functions but then rename it `createForAbapOnBtpWithReentranceTicket`

Reply:
yes